collect_app/src/main/res/layout/range_widget_horizontal.xml

```diff
@@ -31,16 +34,15 @@ limitations under the License.
         android:layout_width="match_parent"
         android:layout_height="wrap_content">
 
-        <SeekBar
-            android:id="@+id/seek_bar_no_ticks"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content" />
-
-        <SeekBar
-            android:id="@+id/seek_bar"
+        <org.odk.collect.android.views.TrackingTouchSlider
+            android:id="@+id/slider"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:theme="@style/Base.Widget.AppCompat.SeekBar.Discrete" />
+            app:thumbColor="?colorSecondary"
+            app:tickColor="?colorOnBackground"
+            app:haloColor="?dividerColor"
```

Comment:
I don't think we should use `dividerColor` here. Could you find a color to use in the standard [material color set](https://material.io/develop/android/theming/color) and use that instead? If it isn't in our themes already you might have to define it. 

If it seems doable you could then replace `dividerColor` everywhere and delete it? No worries if that seems like a distraction though.

Reply:
Yes sure